### ISSUE TITLE
when using the netcdf output checkmark the following error occurs:

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-results-analysis#557]
with these settings:
![image](https://user-images.githubusercontent.com/1453159/122574862-66777d00-d050-11eb-8b3d-baf2eb102dfe.png)

![image](https://user-images.githubusercontent.com/1453159/122574667-2f08d080-d050-11eb-9899-87e67c7e99d6.png)

```
QGIS version: 3.16.1-Hannover
QGIS code revision: b381a90dca
Qt version: 5.11.2
GDAL version: 3.1.4
GEOS version: 3.8.1-CAPI-1.13.3
PROJ version: Rel. 6.3.2, May 1st, 2020
Processing algorithm…
Algorithm 'Water depth' starting…
Input parameters:
{ 'AS_NETCDF_INPUT' : True, 'CALCULATION_STEP_END_INPUT' : 42, 'CALCULATION_STEP_INPUT' : 0, 'DEM_INPUT' : 'C:/modellen/noordpolder_riolering/rasters/dem_nprijswijk_hybride.tif', 'GRIDADMIN_INPUT' : 'C:\\modellen\\noordpolder_riolering\\results\\noordpolder_testwaterdepth_calculations_jonas\\gridadmin.h5', 'MODE_INPUT' : 0, 'RESULTS_3DI_INPUT' : 'C:\\modellen\\noordpolder_riolering\\results\\noordpolder_testwaterdepth_calculations_jonas\\results_3di.nc', 'WATER_DEPTH_OUTPUT' : 'TEMPORARY_OUTPUT' }

Traceback (most recent call last):
File "C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ThreeDiToolbox\processing\threedidepth_algorithm.py", line 327, in processAlgorithm
netcdf=parameters[self.AS_NETCDF_INPUT],
File "C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\default/python\threedidepth\calculate.py", line 668, in calculate_waterdepth
with converter_class(**converter_kwargs) as converter:
File "C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\default/python\threedidepth\calculate.py", line 455, in __init__
raise ImportError("Could not import netCDF4")
ImportError: Could not import netCDF4

Execution failed after 0.02 seconds

Loading resulting layers
Algorithm 'Water depth' finished`
```

Also it is not really clear to me in the design what the output netcdf name will be and where it will be saved. Am I missing something?

**Comment by reinout** [comment]
Hmmmmm. The changelog for 1.7 (2019-02-28) says "Get rid of NetCDF4 lib".... So apparently Richard's change is older than that? 

I don't know really anything about those various netcdf/h5 formats...

**Comment by Jonasvsl** [comment]
No Richards is newer I suspect he reintroduced the dependency. Makes sense as the purpose of the tool is to write to a NetCDF file

**Comment by reinout** [comment]
Fixed in #559, I hope.
